lib/web_ui/flutter_js/src/browser_environment.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const isBlink = () => {
+  return (navigator.vendor === 'Google Inc.') ||
```

Comment:
This is very nitpicky, and can be done later, but we should pick a quote style and stick to it; either `'` or `"` (line 25) ((back-quotes are fine, I kinda prefer `"` for JS code))

Reply:
Yes sorry I tried to be consistent with double quotes in JS code but missed some spots.